subprojects/androidTest/src/androidTest/java/org/mockitousage/androidtest/BasicInstrumentedTests.kt

```diff
@@ -35,7 +35,6 @@ class BasicInstrumentedTests {
     fun mockAndUseBasicClassUsingAnnotatedMock() {
         val basicClass = BasicOpenClassReceiver(mockedViaAnnotationBasicOpenClass)
         basicClass.callDependencyMethod()
-        throw Exception("java")
```

Comment:
This test seemed to be not valid: the exception was thrown but no validations / versification or what not

Reply:
Agree, it was probably testing failure, because a CI job is as good as how you can diagnose failures.